<?xml version="1.0" encoding="UTF-8"?>

<launch>
 <!-- recommend : launch is not that recommended. rather rosrun in each window for monitoring-->

  <!-- vicon -->
  <include file="$(find vicon_bridge)/launch/vicon.launch"/>
  
  <!--ZED-->
  <include file="$(find zed_wrapper)/launch/display.launch"/>
  
  <!--tf connecting btw vicon and zed camera-->
  <node name="connecting_tf" pkg="zed_tracking" type="connecting_tf_vicon_zed" output="screen"/>
  
  <!--HSV tracking -->
  <node name="HSV_tracking_pos_estimate" pkg="zed_tracking" type="HSV_tracking" output="screen">
  		<param name="H_max" value="76" type="int" />
  		<param name="H_min" value="31" type="int" />
  		<param name="S_max" value="255" type="int" />
  		<param name="S_min" value="116" type="int" />
  		<param name="V_max" value="255" type="int" />
  		<param name="V_min" value="0" type="int" />
  </node>
  
</launch>
  
